### DISCUSSION TITLE
time for scheduled option as days or weeks

### DISCUSSION THREAD

**Discussion by saiballo** [borgbase/vorta#1577] [General]
If I select a scheduled periodic backup of "3 days" or "1 weeks", the backup is scheduled correctly for the day but the time is ever 00:00, when my pc is power off.

How can I change the time for scheduled backup? (non every day of course)

**Comment by real-yfprojects** [comment]
> How can I change the time for scheduled backup? (non every day of course)

So far You can't. This already tracked in #1403. If configured vorta will do the backup once your computer starts. Doesn't this option satisfy your use case?
